docs/site/Validation.md

```diff
@@ -0,0 +1,28 @@
+---
+lang: en
+title: 'Validation'
+keywords: LoopBack 4.0, LoopBack 4
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Validation.html
+---
+
+Within a LoopBack application, validation can be added in various places
+depending on the usage. Some types of validations come out-of-the-box in
+LoopBack, such as type validation in the REST layer, whereas some require
+additional configuration or code.
+
+For illustration purpose, a `CoffeeShop` model is being used. It has the
+following properties.
+
+| Property name |  Type  | Description                           |
+| ------------- | :----: | ------------------------------------- |
+| shopId        | string | ID of the coffee shop                 |
+| city          | string | City where the coffee shop is located |
+| phoneNum      | string | Phone number of the coffee shop       |
+| capacity      | number | Capacity of the coffee shop           |
+
+Let's take a closer look at how validation can be added in the following layers:
+
+- [REST layer](Validation-REST-layer.md)
+- [Controller layer](Validation-controller-layer.md)
+- [ORM layer](Validation-ORM-layer.md)
```

Comment:
In addition to layers, we should also talk about different types of validations such as:

- Validation of input/output for method invocations
- Validation of model instance properties (for example, age > 0)
- Validation of model collections (for example, uniqueness)

Reply:
if that's the case, where do you think we should add it to? out of the 3 docs page?

Reply:
Maybe describe the use cases at the index page level.